pyrightconfig.testcases.json

```
@@ -0,0 +1,23 @@
+{
+    "$schema": "https://raw.githubusercontent.com/microsoft/pyright/main/packages/vscode-pyright/schemas/pyrightconfig.schema.json",
+    "include": [
+        "tests/assert_type/"
+    ],
+    "typeCheckingMode": "strict",
+    // Extra strict settings
+    "reportShadowedImports": "error", // Don't accidentally name a file something that shadows stdlib
+    "reportImplicitStringConcatenation": "error",
+    "reportUninitializedInstanceVariable": "error",
+    "reportUnnecessaryTypeIgnoreComment": "error",
+    // Using unspecific `type: ignore` comments in test_cases.
+    "enableTypeIgnoreComments": true,
```

Should we turn off `enableTypeIgnoreComments` and instead work with `pyright: ignore[...]` for pyright? That way dedicate any `type: ignore[...]` to `mypy`

Would the above work, or am I missing something?

```suggestion
    // Don't use '# type: ignore' to suppress with pyright
    "enableTypeIgnoreComments": false,
```

---

Seems to work well!